lib/ansible/galaxy/api.py

```diff
@@ -46,6 +47,27 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def rate_limit_backoff(function):
+    def run_function(*args, **kwargs):
+        # TODO: support user configuration
+        retries = 6
+        delay = 2
+        max_delay = 40
+
+        for retry in range(0, retries):
+            delay = random.randint(0, min(max_delay, delay * 2 ** retry))
+
+            try:
+                return function(*args, **kwargs)
+            except GalaxyError as e:
+                # Galaxy can't always return a helpful error code because cloud.redhat.com masks it
```

Comment:
I'm not sure what this comment tries to explain. How does this problem manifest itself in our runtime?